qmri_vfa/VFA.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MagneticFieldStrength": "3",
+  "Manufacturer": "Siemens",
+  "ManufacturerModelName": "TrioTim",
+  "PulseSequenceType": "SPGR",
+  "RepetitionTime": "0.0150"
```

Comment:
Similar to MTS, `RepetitionTime` may be different. Do we need to make some notes about this, or is it clear to the users (somewhere in the spec and maybe even at the validation level) that the parameters are listed under common file only when they are constant across file collection pairs?

Reply:
Will cross check. Not sure about the spec or the validator about this.
Best way to quickly know for the validator is to stress test it.